plotjuggler_plugins/ParserROS/rosx_introspection/CMakeLists.txt

```diff
@@ -9,15 +9,6 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 
-find_package(fastcdr QUIET)
-
-if(NOT fastcdr_FOUND )
-    # Override Fast-CDR option: compile as static lib
-    SET(BUILD_SHARED_LIBS OFF CACHE BOOL "Create shared libraries by default")
-    add_subdirectory(3rdparty/Fast-CDR)
-    include_directories(3rdparty/Fast-CDR/include)
-endif()
-
```

Comment:
I don't understand. If we are not compiling the library in 3rd party, how this even work?

Reply:
I moved it to the upper makefile:
plotjuggler_plugins/ParserROS/CMakeLists.txt

Reply:
Yes, I believe you should move that code back to the other Cmake, thanks!!!

Reply:
Moved back 3rdparty build where it was.
had to leave find(fastcdr QUIET) though otherwise build would fail with fastcdr installed on build machine i.e
this issue: https://github.com/facontidavide/PlotJuggler/issues/861

Also rebased on latest main which is why (I think) windows CI does not pass.